pkg/modeling/modeling.go

```diff
@@ -63,7 +63,10 @@ type ResourceList map[clusterapis.ResourceName]resource.Quantity
 
 // InitSummary is the init function of modeling data structure
 func InitSummary(resourceModels []clusterapis.ResourceModel) (ResourceSummary, error) {
-	rsName, rsList := []clusterapis.ResourceName{}, []ResourceList{}
+	var (
+		rsName []clusterapis.ResourceName
+		rsList []ResourceList
+	)
```

Comment:
```suggestion
	var rsName []clusterapis.ResourceName
	var rsList []ResourceList
```
We don't usually use variable blocks.

Reply:
Got it.